pkg/netrc/reader.go

```diff
@@ -42,23 +43,23 @@ func parseNetrc(data string) []netrcLine {
 		i := 0
 		for ; i < len(f)-1; i += 2 {
 			// Reset at each "machine" token.
-			// “The auto-login process searches the .netrc file for a machine token
+			// "The auto-login process searches the .netrc file for a machine token
 			// that matches […]. Once a match is made, the subsequent .netrc tokens
 			// are processed, stopping when the end of file is reached or another
-			// machine or a default token is encountered.”
+			// machine or a default token is encountered."
 			switch f[i] {
 			case "machine":
 				l = netrcLine{machine: f[i+1]}
 			case "default":
-				break
+				break lines
```

Comment:
[The spec](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html) says:

```
‘default’
This is the same as machine name except that default matches any name. 
There can be only one default token, and it must be after all machine tokens. 
```

In this context, the break statement is intended to exit only the inner loop so that we can evaluate whether the current token corresponds to the `default` case and then decide whether to continue iterating the outer loop. 

Can we just ignore the linting err  with `//nolint:staticcheck` directive for this case?